x/stake/keeper/distribute_sidechain.go

```diff
@@ -264,8 +280,26 @@ func (k Keeper) distributeSingleBatch(ctx sdk.Context, sideChainId string) {
 			distAddrBalanceMap[distAddr.String()] = reward.Amount
 		}
 
-		if _, _, err := k.bankKeeper.AddCoins(ctx, reward.AccAddr, sdk.Coins{sdk.NewCoin(bondDenom, reward.Amount)}); err != nil {
-			panic(err)
+		if reward.CrossStake && sdk.IsUpgrade(sdk.BEP153) {
+			rewardCAoB := types.GetStakeCAoB(reward.AccAddr.Bytes(), "Reward")
+			if _, _, err := k.BankKeeper.AddCoins(ctx, rewardCAoB, sdk.Coins{sdk.NewCoin(bondDenom, reward.Amount)}); err != nil {
+				panic(err)
+			}
+			balance := k.BankKeeper.GetCoins(ctx, rewardCAoB).AmountOf(bondDenom)
+			if balance >= 1e8 {
```

Comment:
Define a constant, not use value directly here.

Reply:
Done